core/templates/dev/head/pages/community-dashboard-page/contributions-and-review/contributions-and-review.directive.html

```diff
@@ -0,0 +1,127 @@
+<div class="oppia-contributions-show-review">
+  <div class="oppia-contributions-show-review-side-navbar-container" >
+    <div class="oppia-contributions-side-navbar-part" ng-if="$ctrl.isAdmin">
+      <div class="oppia-contributions-navbar-items-list" navbar-label="Things to do">
+        <div class="oppia-contributions-navbar-item">
+          <button class="oppia-contributions-navbar-button"
+                  ng-click="$ctrl.switchToReviewTab()"
+                  ng-class="$ctrl.reviewTabActive ? 'oppia-contributions-active-navbar': ''"> Review Translations
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="oppia-contributions-side-navbar-part" ng-if="$ctrl.userIsLoggedIn">
+      <div class="oppia-contributions-navbar-items-list" navbar-label="Contributions">
+        <div class="oppia-contributions-navbar-item">
+          <button class="oppia-contributions-navbar-button"
+                  ng-click="$ctrl.switchToContributionsTab()"
+                  ng-class="$ctrl.reviewTabActive ? '' : 'oppia-contributions-active-navbar'">Translations
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-contributions-show-review-container">
+    <div class="oppia-login-required" ng-if="ctrl.userDeatilsLoading && !$ctrl.userIsLoggedIn">
+      <login-required-message></login-required-message>
+    </div>
+    <div class="oppia-login-required">
+      <opportunities-list loading-view="$ctrl.contributionsDataLoading"
+                          opportunities="$ctrl.contributionSummaries"
+                          on-click-action-button="$ctrl.onClickViewSuggestion"
+                          label-required="!$ctrl.reviewTabActive">
+      </opportunities-list>
+    </div>
+  </div>
+</div>
+
+<style>
+  .oppia-contributions-show-review {
```

Comment:
Consider prefixing these rules with `contributions-and-review ` (followed by a space)